Dashboard/Views/StatusView.axaml

```diff
@@ -45,211 +45,399 @@
+                                            <!-- Remote Emergency Stop Status -->
+                                            <StackPanel Grid.Column="2" HorizontalAlignment="Center"
+                                                        VerticalAlignment="Bottom" Grid.Row="0" Grid.RowSpan="2"
+                                                        Margin="20,0,0,0">
+                                                <TextBlock Text="RES Status" FontSize="24" VerticalAlignment="Bottom"
+                                                           Margin="35,0,0,0" />
+                                                <Image Name="ResIcon" Source="../Assets/emergency-stop.png" Width="200"
+                                                       Height="200" Margin="0,0,0,0" Opacity="1" />
+                                                <TextBlock Name="RemoteEmergencyStopIndicator" FontWeight="Bold"
+                                                           Text="{Binding RemoteEmergency}" FontSize="22"
+                                                           Background="Green" Foreground="White" Padding="10,0"
+                                                           Margin="0,0,0,0" TextAlignment="Center"
+                                                           HorizontalAlignment="Center"
+                                                           Width="200" />
+                                            </StackPanel>
+                                        </Grid>
+                                    </StackPanel>
                                 </StackPanel>
-                            </StackPanel>
-                        </Grid>
-                    </TabItem>
+                            </Grid>
+                        </TabItem>
 
-                    <!-- GPS Tab -->
-                    <TabItem Header="GPS">
-                        <StackPanel Width="1200">
-                            <!-- Content for GPS -->
-                            <TextBlock Text="GPS data here" HorizontalAlignment="Center" VerticalAlignment="Center" />
-                        </StackPanel>
-                    </TabItem>
+                        <!-- GPS Tab -->
+                        <TabItem Header="GPS">
+                            <StackPanel Width="1200">
+                                <!-- Content for GPS -->
+                                <Grid ColumnDefinitions="*, *" RowDefinitions="Auto, Auto, Auto, Auto, Auto, Auto, *">
+                                    <StackPanel Grid.Column="0" Grid.Row="0">
+                                        <Grid ColumnDefinitions="Auto, *" Margin="10, 30, 0, 0">
+                                            <TextBlock Grid.Column="0" FontSize="24" Text="SFA:" />
+                                            <TextBlock Text="{Binding SatFixAge}" FontSize="22"
+                                                       Grid.Column="1" VerticalAlignment="Center" Background="White"
```

Comment:
Is there a way to make the white boxes the same width so it looks more uniform? If not all good i know avalonia can be complicated

Reply:
Found a way to keep them the same width :)